r/README.md

```diff
@@ -10,14 +10,25 @@ R integration with Apache Arrow.
 First install a release build of the C++ bindings to arrow.
 
 ``` shell
+# Get source
 git clone https://github.com/apache/arrow.git
-cd arrow/cpp && mkdir release && cd release
+cd arrow
+
+# Create install dir
```

Comment:
these changes to the directory structure are unrelated to this PR, but they're minor and I do think they make the instructions slightly easier to reason about. Let me know if you'd like me to remove them.